updated basic security tools (#1695)
OWASP/owasp-mstg: Document/0x05b-Basic-Security_Testing.md

echo "Want to use it again? Start your emulator with 'emulator -avd NAMEOFX86A8.0 -writable-system -selinux permissive'"
```

> Please note that Xposed, as of early 2020, does not work on Android 9 (API level 28) yet. However it was unofficially ported for Android Pie. You can find the code and usage examples at [EdXposed](https://github.com/ElderDrivers/EdXposed) Github repo.

Reviewer: Included API levels and made a bit more timeless to avoid having to update the year every year :)

```suggestion
Please note that Xposed, at the time of this writing, does not work on Android 9 (API level 28). However, it was unofficially ported in 2019 under the name EdXposed, supporting Android 8-10 (API level 26 till 29). You can find the code and usage examples at [EdXposed](https://github.com/ElderDrivers/EdXposed "EdXposed") Github repo.
```

Before you can start using drozer, you'll also need the drozer agent that runs on the Android device itself. Download the latest drozer agent [from the GitHub releases page](https://github.com/FSecureLABS/drozer/releases/ "drozer GitHub releases") and install it with `adb install drozer.apk`.

> Once the setup is completed you can start a session to an emulator or a device connected per USB by running `adb forward tcp:31415 tcp:31415` and `drozer console connect`. This is called direct mode and you can see the full instructions in the [User Guide in section "Starting a Session"](https://labs.f-secure.com/assets/BlogFiles/mwri-drozer-user-guide-2015-03-23.pdf "Starting a Session"). If for some reason you cannot run Drozer in direct mode you can try to run it in infrastructure mode.  In infrastructure mode, you run a drozer server either on your network or on the Internet that provides a meeting point for your consoles and agents, and routes sessions between them. You can find the details of infrastructure mode in ["Infrastructure Mode"](https://labs.f-secure.com/assets/BlogFiles/mwri-drozer-user-guide-2015-03-23.pdf) section of the User Guide.  

Reviewer 2: ```suggestion
Once the setup is completed you can start a session to an emulator or a device connected per USB by running `adb forward tcp:31415 tcp:31415` and `drozer console connect`. This is called direct mode and you can see the full instructions in the [User Guide in section "Starting a Session"](https://labs.f-secure.com/assets/BlogFiles/mwri-drozer-user-guide-2015-03-23.pdf "Starting a Session"). An alternative is to run Drozer in infrastructure mode, where, you are running a drozer server that can handle multiple consoles and agents, and routes sessions between them. You can find the details of how to setup drozer in this mode in the ["Infrastructure Mode"](https://labs.f-secure.com/assets/BlogFiles/mwri-drozer-user-guide-2015-03-23.pdf "Infrastructure Mode") section of the User Guide.  
```

Reviewer: @sushi2k I'd suggest to put both occurrences of "User Guide" in lowercase.
